test/e2e/topology_updater_test.go

```diff
@@ -261,6 +261,28 @@ var _ = NFDDescribe(Label("nfd-topology-updater"), func() {
 				return true
 			}, time.Minute, 5*time.Second).Should(BeTrue(), "didn't get updated node topology info")
 		})
+
```

Comment:
Maybe also check if proper `ownerReference` was added to the NRT object.

Reply:
@PiotrProkop
Thanks, added a check.